.gitignore

```diff
@@ -43,3 +42,89 @@ Temporary Items
 .nfs*
 
 # End of https://www.gitignore.io/api/macos,linux
+.env
+Icon


```

Comment:
This should not be here, it's already here: https://github.com/ga-wdi-boston/ruby-vs-js-study/pull/52/commits/4e570b6c71ac66dd1e41fcd49336d9b370ed01a0#diff-a084b794bc0759e7a6b77810e01874f2R10